hal/network/ncp_client/esp32/esp32_ncp_client.cpp

```diff
@@ -68,12 +80,18 @@ const auto ESP32_NCP_KEEPALIVE_MAX_MISSED = 5;
 const auto ESP32_NCP_AT_CHANNEL_RX_BUFFER_SIZE = 4096;
 
 const auto ESP32_NCP_DEFAULT_SERIAL_BAUDRATE = 921600;
+const auto ESP32_NCP_DEFAULT_SDIO_SPEED = 8000000;  // 8MHz
 
 const auto ESP32_NCP_AT_CHANNEL = 1;
 const auto ESP32_NCP_STA_CHANNEL = 2;
 const auto ESP32_NCP_AP_CHANNEL = 3;
 
+#if !HAL_PLATFORM_WIFI_NCP_SDIO
 const auto ESP32_NCP_MIN_MVER_WITH_CMUX = 4;
+#else
+// FIXME:
+const auto ESP32_NCP_MIN_MVER_WITH_CMUX = 7;
```

Comment:
7 or 8?

Reply:
We'll go with 7. I should remove the FIXME. The latest (albeit non public) release is 0.0.6. There is no point to jump two version.